sp800-63c/sec5_strength.md

```diff
@@ -74,14 +74,19 @@ All assertions SHOULD use audience restriction techniques to allow an RP to reco
 
 Note that in a proxied federation model, ultimate IdP may be unable to generate a pairwise pseudonymous identifier for the ultimate RP, since the proxy could blind the IdP from knowing which RP is being accessed by the subscriber. In such situations, the pairwise pseudonymous identifier is usually between the IdP and the federation proxy itself. The proxy, acting as an IdP, can itself provide pairwise pseudonymous identifiers to downstream RPs. Depending on the protocol, the federation proxy may need to map the pairwise pseudonymous identifiers back to the associated identifiers from upstream IdPs in order to allow the identity protocol to function. In such cases, the proxy will be able to track and determine which pairwise pseudonymous identifiers represent the same subscriber at different RPs.
 
-#### 5.2.6. Pairwise Pseudonymous Identifier Generation
+#### 5.2.6. <a name="ppi-gen"></a>Pairwise Pseudonymous Identifier Generation
 
-Pairwise pseudonymous identifiers SHALL be opaque and unguessable, containing no identifying information about the subscriber. Additionally, they SHALL only be known and used by one IdP-RP pair. 
+Pairwise pseudonymous identifiers SHALL be opaque and unguessable, containing no identifying information about the subscriber. Additionally, the identifiers SHALL only be known by and used by one IdP-RP pair. 
 
-If several RPs have an explicit and tight correlation, such as a shared security domain or legal ownership, those RPs MAY be combined into a single logical RP for the purposes of generating a pairwise pseudonymous identifier. The IdP SHALL ensure that only intended RPs are in such a group, otherwise a rogue RP could learn of the pairwise pseudonymous identifier by posing as part of an RP group of which it is not a member.
\ No newline at end of file
+An IdP MAY generate the same identifier for a subscriber at multiple RPs at the request of those RPs, but only if:
+
+* Those RPs have a demonstrable relationship that justifies an operational need for the correlation, such as a shared security domain or shared legal ownership, and
+* All RPs consent to being correlated in such a manner.
+
+The RPs SHALL conduct a privacy risk assessment to consider the privacy risks associated with requesting a common identifier. The IdP SHALL ensure that only intended RPs are correlated, otherwise a rogue RP could learn of the pseudonymous identifier for a correlation by fraudulently posing as part that correlation.
```

Comment:
s/part that correlation/part of that correlation/

While you're at it, the comma after "correlated" should be a semicolon.